.travis.yml

```diff
@@ -0,0 +1,89 @@
+language: php
```

Comment:
We should move this changes to another PR. It is not relevant for testing.
See https://github.com/phingofficial/phing-composer-configurator/pull/5